jenkins/scripts/bare_metal_lab/deploy-lab.yaml

```diff
@@ -41,6 +41,14 @@
       become: true
       when: sshd_config.changed
 
+    - name: Shrink size of syslog
+      become: true
+      become_user: root
+      shell:
+        cmd: truncate --size 10737418240 /var/log/syslog && truncate --size 10737418240 /var/log/syslog1   2>/dev/null
```

Comment:
The rotated file is called /var/log/syslog.1, not /var/log/syslog1. How have you tested this command?

Reply:
I changed the type and tested. It is working now. 
This is the resulted syslog size:    10738MB Dec  8 11:07 syslog